client/script/test.bash

```diff
@@ -41,6 +41,6 @@ elif [ "$mode" == "-e" ]; then
     yarn configapp dev
 else
     printf "Running unit tests only...\n"
-    printf "Use -a for all tests, -u for unit tests or -e for e2e tests"
+    printf "Use -a for all tests, -u for unit tests, -i for integration tests or -e for e2e tests"
```

Comment:
Where does the URL for the gitlab instance come for the tests? If it's from config/test.js, there is an implicit assumption of the react website running for the E2E tests. Is this the case? If it is the case, then please make this part of test.bash for only E2E tests.

Reply:
So get rid of `yarn configapp test` and instead just make the e2e use the `config/test.js` in the code. I will work on this change.

Reply:
That's not what I meant. Please keep the `yarn configapp test` and `config/test.js`. I am asking about the code in the existing test.bash that starts the react app before playwright tests it.

Ideally the following things happen in E2E tests of `client.yml`.
```
yarn install
yarn build
yarn configapp test
yarn start &
yarn test -a #all modes - u,i,e,a - should work here
``` 

>I am asking about the code in the existing test.bash that starts the react app before playwright tests it.

Answer to this comment clarifies the commands to be used for starting (`yarn start`)and stopping (we don't have a command/script for this) the react website during E2E tests.

Reply:
`yarn test -a` will not work as the e2e tests will fail, they require GitLab user credentials which I do not want to expose on the github. The command for stopping the react website is `npx kill-port 4000` no?

Reply:
Makes sense with the credentials. Then only unit and integration tests run on github actions. The npx command seems right.